From: David Herrmann <dh.herrmann@gmail.com>
Date: Mon, 16 Mar 2015 10:17:10 +0100
Subject: [PATCH] samples/kdbus: drop wrong include

There is no reason to use ./include/uapi/ directly from samples. If your
system headers are not up-to-date, you _need_ to run "make
headers-install" (which will install them to ./usr/ in your kernel tree)
before building the examples. Otherwise, you will get warnings and build
failures.

Once ./usr/ is updated with the correct headers, it contains everything we
need, so drop -Iinclude/uapi from the kdbus-workers CFLAGS.

Signed-off-by: David Herrmann <dh.herrmann@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 samples/kdbus/Makefile | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/samples/kdbus/Makefile b/samples/kdbus/Makefile
index eee9b9aed632..e714602b6260 100644
--- a/samples/kdbus/Makefile
+++ b/samples/kdbus/Makefile
@@ -5,7 +5,5 @@ hostprogs-y += kdbus-workers
 
 always := $(hostprogs-y)
 
-HOSTCFLAGS_kdbus-workers.o +=		\
-	-I$(objtree)/usr/include/	\
-	-I$(objtree)/include/uapi/
+HOSTCFLAGS_kdbus-workers.o += -I$(objtree)/usr/include
 HOSTLOADLIBES_kdbus-workers := -lrt
